scripts/testnet/include/config.sh

```diff
@@ -146,6 +152,8 @@ updateNodeConfig() {
 	  sed -i '/^\[ElasticSearchConnector\]/,/^\[/ s/Enabled *= *false/Enabled = true/' external_observer.toml
 	fi
 
+  sed -i '/^\[DbLookupExtensions\]/,/^\[/ s/Enabled *= *false/Enabled = true/' config_observer.toml
```

Comment:
I don't think this should be by default set to true.
I think we should use an exported variable for this...

Opinions?
@bogdan-rosianu @raduchis

Reply:
in my opinion, it's ok to be default to true. we can have an exported var for this, but I would like to make it default to true

Reply:
if this will be connected to a proxy and an api, it should be default true.